### ISSUE TITLE
[ALM Accelerator - DOCUMENTATION]: Import Unmanaged Deployment Configuration

### ISSUE THREAD

**Issue by joje1234** [microsoft/coe-starter-kit#4566]
### Describe the issue

I'm trying to import a solution with the ALM app, but in the 'Import Unmanaged Solution' -step in the pipeline, I get this error:
[ 'Error: Environment variable value cannot be an empty string.' ]

The solution have 6 environment variables, but all of them have values in the "Configure deployment settings" in the ALM app, for all environments.

There is no log-file produced in the environment I'm trying to import the solution to.
I can download the artifact, and install the solution manually.

I have tried installing the solution manually, commit and deploy it again, and install it from the ALM app again, but get the same error.

![image](https://user-images.githubusercontent.com/32527403/211812539-a6440836-3818-4a1d-bd73-e0f3cf25843a.png)

### Expected Behavior

Expect the solution to install normally.

### What component are you experiencing the issue with?

ALM Accelerator Pipelines

### What solution version are you using?

 1.0.20221206.2

**Comment by joje1234** [comment]
My intension is just to install the solution as unmanaged from the source code. It normally works, but this bug is new for me.

**Comment by joshuadow** [comment]
This is also happening to me after installing the December2022 version of the templates and ALM.

**Comment by mikefactorial** [comment]
@joshuadow @joje1234 a couple of questions to help with duplicating / troubleshooring.
1. Are you selecting a Configuration source during the import? 
  - If so, does the solution import when the Configuration is blank and does that configuration file contain the correct information about the environment to which you are importing e.g. all environment variables have valid values and aren't empty?
2. Does the solution in question exist in the target environment or is it a blank environment?

**Comment by joshuadow** [comment]
Hi @mikefactorial ! 

I have tried both with it blank and with a configuration selected. When I look at the failed pipeline run, and download the artifacts, all the correct information is there. Including environment variables having all default values in their XML definitions.

I have also tried bringing it into both environments where the solution does exist and where it doesn't.

I was successful in importing a solution that was committed and deployed using the December 2022 release of the ALM. It makes me think there is some sort of difference in the schema that the new ALM remedies on export but does not consider on import.
